src/StaticResourceHandlerFactory.php

```diff
@@ -71,6 +71,7 @@ public function __invoke(ContainerInterface $container): StaticResourceHandler
     {
         $config = $container->get('config')['mezzio-swoole']['swoole-http-server']['static-files'] ?? [];
 
+        /** @psalm-suppress MixedArrayAccess */
```

Comment:
I think these really should go towards our baseline, instead of being suppressed :thinking:

Reply:
Sounds reasonable - I've pushed a new baseline, and reverted the suppressions via a force commit.